From 724309dc3846926f53d58ea8fa981b25b71c7c71 Mon Sep 17 00:00:00 2001
From: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Date: Thu, 28 Feb 2019 05:46:22 -0800
Subject: [PATCH 0526/1851] dt-bindings: ASoC: xlnx,audio-formatter.txt: update
 clocks

commit 99b6418f8cfc53eb9139633d76dcde413ef397d6 from
https://github.com/Xilinx/linux-xlnx.git

updated with "m_axis_mm2s_aclk", "s_axis_s2mm_aclk" clocks.

Signed-off-by: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/sound/xlnx,audio-formatter.txt      | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/sound/xlnx,audio-formatter.txt b/Documentation/devicetree/bindings/sound/xlnx,audio-formatter.txt
index e98ed1706e21..1d255020727d 100644
--- a/Documentation/devicetree/bindings/sound/xlnx,audio-formatter.txt
+++ b/Documentation/devicetree/bindings/sound/xlnx,audio-formatter.txt
@@ -27,7 +27,7 @@ Required properties:
 		3. UHDSDI audio extract
 	    Only those nodes were supported in sound card driver
  - clock-names: List of input clocks.
-   Required elements: "s_axi_lite_aclk", "aud_mclk"
+   Required elements: "s_axi_lite_aclk", "m_axis_mm2s_aclk", "aud_mclk", "s_axis_s2mm_aclk"
  - clocks: Input clock specifier. Refer to common clock bindings.
 
 Example:
@@ -39,6 +39,6 @@ Example:
 		reg = <0x0 0x80010000 0x0 0x1000>;
 		xlnx,tx = <&i2s_transmitter>;
 		xlnx,rx = <&i2s_receiver>;
-		clock-names = "s_axi_lite_aclk", "aud_mclk";
-		clocks = <&clk 71>, <&clk_wiz_1 0>;
+		clock-names = "s_axi_lite_aclk", "m_axis_mm2s_aclk", "aud_mclk", "s_axis_s2mm_aclk";
+		clocks = <&clk 71>, <&audio_ss_0_clk_wiz_0 0>, <&audio_ss_0_clk_wiz_0 0>, <&clk 71>;
 	};
-- 
2.31.1

